sqlbrite/src/main/java/com/squareup/sqlbrite/BriteContentResolver.java

```diff
@@ -120,9 +120,10 @@ public QueryObservable createQuery(@NonNull final Uri uri, @Nullable final Strin
       }
     };
     Observable<Query> queryObservable = Observable.create(subscribe) //
+        .onBackpressureLatest() // Guard against uncontrollable frequency of upstream emissions.
```

Comment:
It doesn't matter. `startWith` only emits one item and then defers items to the upstream observable. The only case where you'd actually get a backpressure problem with a single item is if the queue size was 0, which is impossible.

My next PR will eliminate `startWith` in this case anyway and move it into the upstream.